hedera-node/configuration/README.md

```diff
@@ -43,7 +43,7 @@ path is given, the file belongs in the top-level directory with the JAR.
   - For example, more permissive Netty HTTP/2 settings
 
 ## upgrade/throttles.json
-* If present, should be used, post-upgrade, as input to a yahcli `sysfiles upload throttles` command
+* If present, will be used, post-upgrade, as input to a yahcli `sysfiles upload throttles` command. It will not layer on top of other throttles, it will be exactly what is in this file. If not present it will use https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/resources/throttles.json.
```

Comment:
Default file is under `hedera-file-service-impl/src/main/resources/genesis` for mod service